products/PurePhone/ProductKeyFeatures.md

```diff
@@ -0,0 +1,25 @@
+PurePhone product key features
+==============================
+
+- 3 modes (offline, do not disturb, connected) with magnetometer support
+- approved contacts list
+- threaded messaging with templates, drafts, UTF8 and basic emoji support
+- audio player supporting MP3, WAV, FLAC formats and indexed ID3 Tags
+- meditation app
+- calendar app
+- calculator
+- alarm clock
+- contacts and calendar synchronization via Mudita Center desktop app 
+- flashlight
+- user interface optimized for E Ink displays with frontlight support
+- dark mode
+- Bluetooth headset connectivity with A2DP (Advanced Audio Distribution Profile) and HSP (Headset Profile) support
+- Passive Dual SIM support
+- USB-C Fast Charging
+- VoLTE and internet tethering support
+- HAL (Hardware Abstraction Layer) between the hardware and the OS
+- system manager that restores the system after switching on the device or performs the initial start-up
```

Comment:
what do we restore? XD

Reply:
I believe the author wanted to express that we do not perform a fresh start at each boot and nonvolatile settings are loaded (i.e. dark screen mode). But it's just my assumption I suppose someone more business savvy should tell us how should we introduce products the smart way...